<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>624</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Monaco</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>EP lab_2</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(224, 212, 255);</string>
  </property>
  <widget class="QPushButton" name="calcButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>240</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_req_cnt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>357</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Время моделирования   </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_time">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1710</x>
     <y>630</y>
     <width>1811</width>
     <height>171</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>90</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>№ опыта</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>x0</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>x1</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>x2</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string>x3</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="5">
    <property name="text">
     <string>x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="6">
    <property name="text">
     <string>x1x2</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="7">
    <property name="text">
     <string>x1x3</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="8">
    <property name="text">
     <string>x1x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="9">
    <property name="text">
     <string>x2x3</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="10">
    <property name="text">
     <string>x2x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="11">
    <property name="text">
     <string>x3x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="12">
    <property name="text">
     <string>x1x2x3</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="13">
    <property name="text">
     <string>x1x2x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="14">
    <property name="text">
     <string>x1x3x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="15">
    <property name="text">
     <string>x2x3x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="16">
    <property name="text">
     <string>x1x2x3x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="17">
    <property name="text">
     <string>Y</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="18">
    <property name="text">
     <string>Yл</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="19">
    <property name="text">
     <string>Yнл</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="title">
    <string>Точка для проверки</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>201</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_edit_gen_int">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_a_7">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_edit_pm_int">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_a_9">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_edit_gen_var">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="line_edit_pm_var">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_a_12">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_a_13">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>240</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="text">
    <string>Проверить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_b_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Нелинейное уравнение</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="line_edit_nonlin_res">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>1201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_a_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Линейное уравнение</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="line_edit_lin_res">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>1201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="title">
    <string>Обработчик заявок</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="0">
      <widget class="QLabel" name="label_a_3">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">дисперсия </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_edit_min_pm_var">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="line_edit_max_pm_var">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_a_4">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">интенсивность  </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_edit_min_pm_int">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>min</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="line_edit_max_pm_int">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="title">
    <string>Генератор</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>317</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QLabel" name="label_a_2">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">дисперсия</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>max</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="line_edit_max_gen_var">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_edit_min_gen_int">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_b">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">интенсивность  </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>min</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_edit_min_gen_var">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="line_edit_max_gen_int">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2.5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
